### PR TITLE
Add piano in traverse city mi

### PR DESCRIPTION
**Pull request by knownasilya**

### CODE REVIEW

File: pianos.geojson [hunk 1]
```diff
@@ -98,6 +98,20 @@
           38.1824001
         ]
       }
+    },
+    {
+      "type": "Feature",
+      "properties": {
+        "description": "Piano outside business in downtown",
+        "type": "business"
```
Comment (IonicaBizau) [inline pianos.geojson:106]: What does `business` mean? Maybe we will have to add a section about the `type` parameter. 😄

Comment (knownasilya) [reply pianos.geojson:106]: I'm not sure what type of business it is, but I knew it wasn't a shop or a restaurant.

Comment (IonicaBizau) [reply pianos.geojson:106]: Ah, ok! 😁

Comment (IonicaBizau) [reply pianos.geojson:106]: Thanks!!!

**Pull request merged**